packages/services/api/src/modules/shared/providers/redis.ts

```diff
@@ -5,7 +5,9 @@ import { Logger } from './logger';
 
 export type { RedisInstance as Redis };
 
-export type RedisConfig = Required<Pick<RedisOptions, 'host' | 'port' | 'password'>>;
+export type RedisConfig = Required<Pick<RedisOptions, 'host' | 'port' | 'password'>> & {
+  tls_enabled: boolean;
```

Comment:
Please use camel case instead of snake case.

Reply:
fixed